### PR TITLE
December blog post

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

**Comment by tbrisker** [comment]
Please wait until Wed. Dec 9 before merging to allow time for comments.

File: _posts/2015-12-09-foreman-community-newsletter-december-2015.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+-   Lukas Zapletal (lzap) gave a talk about Foreman at the Pilsen University. Read his report [here](https://groups.google.com/forum/#!topic/foreman-dev/l3-4a6pQTZU). Thanks Lukas!
+-   Daniel Lobato Garcia (dlobatog) gave a talk about [Rails security](https://speakerdeck.com/elobato/20-actionable-tips-to-secure-your-rails-application-rails-israel-15) at the Rails Israel conference. He also joined the Israeli Foreman team members to show off the project at the Red Hat booth.
+
+### Upcoming events
+
+-   This Thursday (Dec. 10th) is community demo time again.
+    Check out the [Latest Demo Information](http://projects.theforeman.org/projects/foreman/wiki/Current_Sprint_Information) page for agenda and viewing links.
+-   The CFP for [FOSDEM](https://fosdem.org/) and [Config Management Camp](http://cfgmgmtcamp.eu/) has closed.
+    We are awaiting the agenda to be announced, hopefully with many Foreman related talks.
+    Lookin forward to seeing you all in Belgium on January 30 - February 2 , 2016.
```
Comment (ohadlevy) [inline _posts/2015-12-09-foreman-community-newsletter-december-2015.md]: nitpick: Looking

File: _posts/2015-12-09-foreman-community-newsletter-december-2015.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+-   [foreman\_openscap](https://github.com/theforeman/foreman_openscap) and
+    [smart\_proxy\_openscap](https://github.com/theforeman/smart_proxy_openscap)
+    0.5.0 was released Novemeber 1st. Please note, this version is only compatible with Foreman 1.11 (current develop branch).
+-   [foreman\_remote\_execution](https://github.com/theforeman/foreman_remote_execution)
+    0.1.1 was released November 12th.
+-   [foreman-xen](https://github.com/theforeman/foreman-xen)
+    0.2.3 (for Foreman >= 1.10) and 0.1.7 (for older versions) were released Novemeber 17th.
+-   [foreman-tasks](https://github.com/theforeman/foreman-tasks)
+    0.7.7 was released December 1st.
+
```
Comment (GregSutcliffe) [inline _posts/2015-12-09-foreman-community-newsletter-december-2015.md:45]: foreman-templates 2.0.1 - fixes an important bug in template prefixes

File: _posts/2015-12-09-foreman-community-newsletter-december-2015.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+- foreman
+- katello
+- newsletter
+- openscap
+- xen
+- remote execution
+- high availability
+- windows imaging
+excerpt: |
+  Foreman 1.10 and Katello 2.4 coming soon
```
Comment (GregSutcliffe) [inline _posts/2015-12-09-foreman-community-newsletter-december-2015.md]: This is quite dry - remember that people will read this before deciding to click through to the post. Maybe mention the new handbook, and the case study/deep dive updates?

File: _posts/2015-12-09-foreman-community-newsletter-december-2015.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    0.1.1 was released November 12th.
+-   [foreman-xen](https://github.com/theforeman/foreman-xen)
+    0.2.3 (for Foreman >= 1.10) and 0.1.7 (for older versions) were released Novemeber 17th.
+-   [foreman-tasks](https://github.com/theforeman/foreman-tasks)
+    0.7.7 was released December 1st.
+
+### Other Updates
+
+-   A new [feature](https://groups.google.com/forum/#!topic/foreman-users/3RYfo0S5c8M) is planned to provide more information about smart proxies in Foreman, we would be happy to hear your input on this matter.
+-   An initial [Developer Handbook](http://theforeman.org/handbook.html) has been created to ease the entry of new contributers and help maintain high standards of existing ones.
```
Comment (GregSutcliffe) [inline _posts/2015-12-09-foreman-community-newsletter-december-2015.md:48]: This is huge, put it first on the list ;)

File: _posts/2015-12-09-foreman-community-newsletter-december-2015.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+    Feel free to fork and open a pull request if you feel some important information is missing!
+-   The Foreman installer has been updated to drop puppet 2.7 support starting with the next version of Foreman (1.11).
+    For more information please read the [announcement](https://groups.google.com/forum/#!topic/foreman-users/YQNfxUzG_As).
+-   The development infrastructure has seen some updates - Redmine has been updated to version 2.6.8 and Jenkins has been updated to version 1.625.2.
```
Comment (GregSutcliffe) [inline _posts/2015-12-09-foreman-community-newsletter-december-2015.md]: A mention of the review-culture plan thread and/or blog post would be good, I think.

File: _posts/2015-12-09-foreman-community-newsletter-december-2015.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+-   The CFP for [FOSDEM](https://fosdem.org/) and [Config Management Camp](http://cfgmgmtcamp.eu/) has closed.
+    We are awaiting the agenda to be announced, hopefully with many Foreman related talks.
+    Lookin forward to seeing you all in Belgium on January 30 - February 2 , 2016.
+    There has been a proposal to have an extra Foreman Hackday on February 3rd, right after Config Management camp.
+    If you are interested, please fill out this extra short [survey](https://docs.google.com/forms/d/1LI5-L7sgWoZqFvph_hUgkNOEcH0fZ5pRyrGYFqS6DDs/viewform).
+
+You can follow us on Twitter: [@ForemanProject](https://twitter.com/ForemanProject).
+If you tweet about foreman, please use the hashtag [\#theforeman](https://twitter.com/hashtag/theforeman).
+
+We also have a [Facebook group](https://www.facebook.com/groups/theforeman/) which you can join for updates & discussion.
```
Comment (GregSutcliffe) [inline _posts/2015-12-09-foreman-community-newsletter-december-2015.md]: Social media info is automatically added in a footer of the blog post layout, no need for you to add it :)

File: _posts/2015-12-09-foreman-community-newsletter-december-2015.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+    - compliance reports for foreman_openscap (oprazak) - https://www.youtu.be/RCqwTtIXRSw?t=2201
+    - community announcements (gwmngilfen) - https://www.youtu.be/RCqwTtIXRSw?t=2359
+-   We held a deep dive about [Windows Imaging](https://www.youtube.com/watch?v=_R8-0KAkoPc) with Daniel Helgenberger. Thanks Daniel!
+-   We continuted our user case study series with Chris "discr33t" Pisano talking about his [High Availability](https://www.youtube.com/watch?v=bJ5a67SSA-s) setup with Foreman. Thanks Chris!
+-   Lukas Zapletal (lzap) gave a talk about Foreman at the Pilsen University. Read his report [here](https://groups.google.com/forum/#!topic/foreman-dev/l3-4a6pQTZU). Thanks Lukas!
+-   Daniel Lobato Garcia (dlobatog) gave a talk about [Rails security](https://speakerdeck.com/elobato/20-actionable-tips-to-secure-your-rails-application-rails-israel-15) at the Rails Israel conference. He also joined the Israeli Foreman team members to show off the project at the Red Hat booth.
+
+### Upcoming events
+
+-   This Thursday (Dec. 10th) is community demo time again.
```
Comment (GregSutcliffe) [inline _posts/2015-12-09-foreman-community-newsletter-december-2015.md]: Also a new deep dive posted for next Monday - see https://plus.google.com/events/chffgr6g21a56ef3oggt367sq84

**Comment by GregSutcliffe** [comment]
@tbrisker looks good, just a few small extras :)

File: _posts/2015-12-09-foreman-community-newsletter-december-2015.md (hunk 7 above)
Comment (tbrisker) [inline _posts/2015-12-09-foreman-community-newsletter-december-2015.md]: @GregSutcliffe I added this, but I couldn't find any announcement in the mailing lists.

**Comment by tbrisker** [comment]
@GregSutcliffe @ohadlevy addressed your comments.

File: _posts/2015-12-09-foreman-community-newsletter-december-2015.md (hunk 7 above)
Comment (GregSutcliffe) [inline _posts/2015-12-09-foreman-community-newsletter-december-2015.md]: @tbrisker gah, sorry, and thanks for the heads up - https://groups.google.com/d/topic/foreman-users/Vk-4gukujgk/discussion

File: _posts/2015-12-09-foreman-community-newsletter-december-2015.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+### Upcoming events
+
+-   This Thursday (December 10th) is community demo time again.
+    Check out the [Latest Demo Information](http://projects.theforeman.org/projects/foreman/wiki/Current_Sprint_Information) page for agenda and viewing links.
+-   Another Deep Dive will take place on Monday December 14th, where Daniel Lobato will be talking about [Taxonomies](https://plus.google.com/events/chffgr6g21a56ef3oggt367sq84).
+-   The CFP for [FOSDEM](https://fosdem.org/) and [Config Management Camp](http://cfgmgmtcamp.eu/) has closed.
+    We are awaiting the agenda to be announced, hopefully with many Foreman related talks.
+    Looking forward to seeing you all in Belgium on January 30 - February 2, 2016.
+    There has been a proposal to have an extra Foreman Hackday on February 3rd, right after Config Management camp.
+    If you are interested, please fill out this extra short [survey](https://docs.google.com/forms/d/1LI5-L7sgWoZqFvph_hUgkNOEcH0fZ5pRyrGYFqS6DDs/viewform).
```
Comment (GregSutcliffe) [inline _posts/2015-12-09-foreman-community-newsletter-december-2015.md]: This is confirmed as of today! Eventbrite event will be posted to the mailing in a day or two.

**Comment by dLobatog** [comment]
@tbrisker Mind to add a small section mentioning that foreman_ansible now:
- measures playbook runtime
- supports importing nested facts
- callback supports SSL
- is nearing an official 1.0 release :smile:

**Comment by GregSutcliffe** [comment]
Merged as 65253a7 with two very minor typos fixed :) Thanks @tbrisker!

**Pull request closed without merging**